database.php

```diff
@@ -0,0 +1,59 @@
+
+<?php
+
+/**
+ * connect to database
+ *
+ * @return instance
+ */
+
+function connectDataBase (){
+    $db = new PDO('mysql:host=db; dbname=cocktails', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+$connect = connectDataBase();
+
+$get = "SELECT * FROM `cocktails`";
```

Comment:
Avoid doing `SELECT *` in any kind of final code. Write out the specific columns that you are querying as this can have seriously unexpected side effects down the line as your DB grows!